ferrocene/ci/scripts/emulated-aarch64-test-runner.sh

```diff
@@ -74,17 +50,8 @@ cmd_run() {
     sudo update-binfmts --import
     sudo update-binfmts --enable qemu-aarch64
 
-    # We pass --sequential because we've seen deadlocks when running UI tests
-    # without that flag. Test execution will be slower, but at least it won't
-    # lock CI up.
-    echo "===> starting remote-test-server"
-    sudo chroot "${rootfs}" /usr/bin/qemu-aarch64-static /usr/bin/remote-test-server -v --bind 127.0.0.1:12345 --sequential
-}
-
-cleanup_mount() {
-    echo
-    echo "===> unmounting /dev from the rootfs"
-    sudo umount "${rootfs}/dev"
+    export QEMU_CPU=cortex-a53
+    qemu-aarch64-static $emulatordir/remote-test-server -v --bind 127.0.0.1:12345
```

Comment:
Oh!!! We don't need `--sequential` anymore? Nice.

Reply:
that's right. multi-threading is now possible. I removed a `--test-threads=1` argument that was only being applied to this target from `src/bootstrap/src/core/build_steps/test.rs`